README.rst

```diff
@@ -50,6 +50,40 @@ Current features
 * **template tags**: tag templates to automate different types of auto-configurations (eg: mesh, WDS, 4G)
 * **simple HTTP resources**: allow devices to automatically download configuration updates
 * **VPN management**: easily create VPN servers and clients
+* **Template variables**: makes it possible to declare context (configuration variables) in template configuration while setting the default values for these declared context in the default values field to bypass validation.
```

Comment:
@NoumbissiValere 
Should we move the focus of this line from `bypass validation` to `allows setting variable`; i.e from user's perspective.

P.S: I removed some changes that I made wouldn't required now. Please rebase again. :smile:

Reply:
Ok @atb00ker :smile: